src/game/stage/entities_manager.c

```diff
@@ -1028,8 +996,8 @@ NONMATCH("asm/non_matching/sub_8008DCC.inc", void sub_8008DCC())
                     regionY++;
                 }
             }
-            em->unkC = gCamera.x;
-            em->unk10 = gCamera.y;
+            em->camX = gCamera.x;
```

Comment:
prevCamX? I think this is used so that it only checks the section between movement rather than the whole viewport every frame

Reply:
Might change it to deltaCamX/Y.
That'd be closer to that, right?